zealot/src/zjson.ts

```diff
@@ -1,7 +1,7 @@
 export interface Item {
   id: number
   schema?: Schema
-  aliases?: string[]
+  aliases?: PrimitiveColumn[]
```

Comment:
The change and the rest were just incomplete typings of the zjson responses from zqd. The stubs that I copied and pasted found these.